packages/lesswrong/server/migrations/2020-05-19-fillDefaultNoIndex.ts

```diff
@@ -0,0 +1,14 @@
+import { registerMigration, fillDefaultValues } from './migrationUtils';
+import { Posts } from '../../lib/collections/posts/collection';
+
+registerMigration({
+  name: "fillDefaultNoIndex",
+  dateWritten: "2020-05-19",
+  idempotent: true,
+  action: async () => {
+    await fillDefaultValues({
+      collection: Posts,
+      fieldName: "noIndex"
```

Comment:
Should this actually specify a value? I assume this creates a `noIndex` key with a value of `null`, seems like it should be explicitly set to `false`.

Reply:
It fills it in with False because that's the default set in the schema declaration.